Source/Core/Core/Movie.cpp

```diff
@@ -1432,7 +1426,7 @@ void SaveRecording(const std::string& filename)
   header.recordingStartTime = s_recordingStartTime;
 
   header.bSaveConfig = true;
-  header.bSkipIdle = s_bSkipIdle;
+  header.bSkipIdle = true;
```

Comment:
hm, forcing it to true here is fine, but what about loading old things which have bSkipIdle=false ? Won't the newer core be unable to play them with the same settings?